settings.py

```diff
@@ -25,4 +25,8 @@
 
 INSTALLED_APPS = [
     "db",
+    "django.contrib.auth",
+    "django.contrib.contenttypes"
 ]
+
+AUTH_USER_MODEL = "db.User"
```

Comment:
The AUTH_USER_MODEL setting should use the get_user_model() function instead of directly referencing the User model. This is a best practice to ensure compatibility with custom user models.